packages/plugin-commands-installation/src/install.ts

```diff
@@ -182,6 +183,10 @@ by any dependencies, so it is an emulation of a flat node_modules',
             description: 'Only allows installation with a store server. If no store server is running, installation will fail',
             name: '--use-running-store-server',
           },
+          {
+            description: 'Clones/hardlinks or copies packages only to virtual store, but not link them to node_modules',
```

Comment:
Why does it not link them to node_modules? What would be the benefit of that? Also, the current implementation does link them to node_modules.

Reply:
My attempt was to let pnpm able to install packages without packages.json, so that docker build process becomes more cache-friendly. And I tested with such condition, through description maybe not correct when package.json was present and linking will actually happen.